src/retina/kernels/abstractkernelengine.h

```diff
@@ -15,6 +15,8 @@ class AbstractKernelEngine: public NeuronifyObject
     Q_OBJECT
     Q_PROPERTY(int resolutionHeight READ resolutionHeight WRITE setResolutionHeight NOTIFY resolutionHeightChanged)
     Q_PROPERTY(int resolutionWidth READ resolutionWidth WRITE setResolutionWidth NOTIFY resolutionWidthChanged)
+    Q_PROPERTY(double shift_x READ shift_x WRITE setShift_x NOTIFY shift_xChanged)
```

Comment:
Could you change `shift_x` to `shiftX` or `xShift` so it follows the naming convention? This also applies to all the usages, such as `emit shift_yChanged`, which should be `emit shiftYChanged` or `emit yShiftChanged`. Also, i think "offset" is a more common term than "shift".

Reply:
fixed in https://github.com/CINPLA/neuronify/pull/180/commits/b762260cae079c931ad833d22c3ef0eacdd5b5c8